README.md

```diff
@@ -26,15 +26,18 @@ Find out more at [wagtail.io](https://wagtail.io/).
 
 ### Getting started
 
-Wagtail works with Python 3, on any platform.
-
-```
-    pip install wagtail
-    wagtail start mysite
-    cd mysite
-    python manage.py migrate
-    python manage.py createsuperuser
-    python manage.py runserver
+Wagtail works with [Python 3](https://www.python.org/downloads/), on any platform.
+
+For quick setup, run the following in a virtual environment:
```

Comment:
I'm not sure about 'For quick setup'. I prefer 'To try out Wagtail' or 'To get started with Wagtail'.

Reply:
Sounds reasonable to me!

Reply:
Updated with the latter: https://github.com/wagtail/wagtail/pull/5210/files#diff-04c6e90faac2675aa89e2176d2eec7d8R31